.changeset/six-coats-know.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
If we don't need to release anything from this PR, we can omit the generating the changeset file, correct?